docs/docs/policies.mdx

```diff
@@ -396,6 +398,308 @@ for details.
+ with a confidence of `1.00`.
+
+`IntentTEDPolicy` should be viewed as an aid for `TEDPolicy`. Since, `TEDPolicy` is expected to improve
+with better coverage of unique conversation paths that the assistant is expected to handle in the training data,
+`IntentTEDPolicy` helps to surface these unique conversation paths from past conversations. For example, if you had
+the following story in your training data:
+
+```yaml
+- story: book_restaurant_table
+  steps:
+  - intent: request_restaurant
+  - action: restaurant_form
+  - active_loop: restaurant_form
+  - action: restaurant_form
+  - active_loop: null
+  - slot_was_set:
+    - requested_slot: null
+```
+
+but an actual conversation might encounter interjections inside the form which you haven't accounted for:
+
+```yaml {10-13}
+- story: actual_conversation
+  steps:
+  - user: |
+        I'm looking for a restaurant.
+    intent: request_restaurant
+  - action: restaurant_form
+  - active_loop: restaurant_form
+  - slot_was_set:
+    - requested_slot: cuisine
+  - user: |
+        Does it matter? I want to be quick.
+    intent: deny
+```
+
+As soon as the `deny` intent gets triggered, the policy handling the form will keep requesting for the `cuisine` slot
+to be filled, as the training stories don't say that this case should be treated differently.
+To help you identify that a special story that handles the user's `deny` intent might be missing at this point,
+ `IntentTEDPolicy` can trigger an `action_unlikely_intent` action right after `deny` intent.
+ Subsequently, you can improve your assistant by adding a new training story that handles this particular case.
+
+To reduce false warnings, `IntentTEDPolicy` has two mechanisms in place at inference time:
+
+1. `IntentTEDPolicy`'s [priority](./policies.mdx#policy-priority) is intentionally kept lower than all
+[rule based policies](./policies.mdx#rule-based-policies). This ensures that if an `action_unlikely_intent`
+is predicted for the last intent, but there is a rule that can be triggered to predict
+the next action, the ensemble of policies is not affected by it and the winning prediction is
+still the one made by `RulePolicy`.
+
+2. `IntentTEDPolicy` does not predict an `action_unlikely_intent` if the last predicted intent
+isn't present in any of the training stories, which might happen if an intent is only used in rules.
+
+As mentioned earlier, while `TEDPolicy` learns the next best action, `IntentTEDPolicy` learns the
+next likely intent that a user can express. Given a conversation context, there can only be a single correct action,
+whereas there can be multiple intents that are likely to occur in the same conversation context. Hence,
+`IntentTEDPolicy` uses a multi-label cross entropy loss function during training. Moreover, it uses
+the [`IntentMaxHistoryTrackerFeaturizer`](#3-intent-max-history) to treat intents as target labels instead of assistant actions or utterances.
```

Comment:
Do we need to say this in the docs?

Reply:
I would be in favor of dropping it, as it potentially adds confusion about IntentTEDPolicy predicting intents vs triggering actions.

Reply:
Me too. It breaks the flow for the reader. I think it is very good information, but maybe put somewhere else, for those who want to learn more about the internals.

Reply:
Dropped it 👍